core_contracts/governance/governance.py

```diff
@@ -91,11 +112,6 @@ def _set_launch_time(self, _day: int) -> None:
     def getLaunchTime(self) -> int:
         return self._launch_time.get()
 
-    @external(readonly=True)
```

Comment:
Aren't there any other scores having dependency on this method?

Reply:
No dependence on that. It is for informational purposes only. The time offset is set in Loans and DEX during the launchBalanced method call.